release-team/role-handbooks/release-notes/README.md

```diff
@@ -66,6 +69,7 @@ beginning of code freeze and the release.
 ## Tools
 
 - [Release notes tool](https://github.com/kubernetes/release/tree/master/cmd/release-notes)
+- [Release notes website](https://relnotes.k8s.io)
 - [Hackmd](https://hackmd.io/)
 - [LWKD](http://lwkd.info) *(consider contributing to LWKD as part of your role)*
 
```

Comment:
Do we want to add the TODOs for 1.15 (or convert the ones from 1.14) as well? :point_down:

Reply:
Yes. I think two of the todos are definitely #668 and #667 and then we should work out the others before this goes live

Reply:
Sounds good, these two should be definitely addressed in the next cycle.